.gitignore

```diff
@@ -29,6 +29,7 @@ rsconnect/
 
 # Data
 inst/doc
+inst/extdata/ccsr_icd10_lookup.csv
```

Comment:
I think this should be done using sysdata as per: https://r-pkgs.org/data.html#data-sysdata - then you shouldn't need to gitignore anything